plugins/techdocs-node/src/stages/generate/types.ts

```diff
@@ -61,6 +62,7 @@ export type GeneratorRunOptions = {
   etag?: string;
   logger: Logger;
   logStream?: Writable;
+  siteName?: string;
```

Comment:
Wondering if we should just wrap this in an options (e.g. `siteOptions: { name: string }`). I can see it being useful to also add `site.description` as well later on.

Reply:
Yes, that makes sense to me. Done 👍